### ISSUE TITLE
Close repo

### ISSUE THREAD

**Issue by jasongrout** [jovyan/gamepad#7]
Since this was merged into ipywidgets (https://github.com/ipython/ipywidgets/pull/123), I think it makes sense to close down this repo.  @SylvainCorlay, what do you think?

**Comment by SylvainCorlay** [comment]
Yes, I was thinking of changing the readme saying that this in deathrow and just there for people using 3.0 < Jupyter <= 4.1...
